### DISCUSSION TITLE
Hosted payment page in iFrame

### DISCUSSION THREAD

**Discussion by timrvv** [impress-org/givewp#7067] [Developer Q&A]
We would like to load gateway's payment hosted page via iFrame so user would not be redirected to separate web-site.
How can we achieve it within GiveWP?

**Comment by JoshuaHungDinh** [comment]
Hey @timrvv, i'm not sure we have a built in solution to accomplish this. It may still be possible, could you please provide additional details on the specific gateway and your planned use case?
